frontend/public/components/factory/list.tsx

```diff
@@ -193,6 +194,13 @@ const listFilters = {
   },
 };
 
+// add filters from resource list page contributions
+plugins.registry.getResourceListPages().forEach(p => {
```

Comment:
See also #1693

@mareklibra fyi

Reply:
Thanks @spadgett. I was not aware of that pr.